android/guava-tests/test/com/google/common/cache/CacheBuilderTest.java

```diff
@@ -502,6 +502,8 @@ public void testRemovalNotification_clear_basher() throws InterruptedException {
     computationShouldWait.set(true);
 
     final AtomicInteger computedCount = new AtomicInteger();
+    // Refactoring this causes test to hang for a while and eventually give wrong result
+    // https://github.com/ponder-lab/guava/actions/runs/9736322153/job/26866789783#step:6:328
```

Comment:
Refactoring this causes [test to hang for a while and eventually give wrong result](https://github.com/ponder-lab/guava/actions/runs/9736322153/job/26866789783#step:6:328)